website/docs/docs/build/incremental-microbatch.md

```diff
@@ -239,6 +240,68 @@ For now, dbt assumes that all values supplied are in UTC:
 
 While we may consider adding support for custom time zones in the future, we also believe that defining these values in UTC makes everyone's lives easier.
 
+## Parallel batch execution
+
+The microbatch strategy offers the benefit of updating a model in smaller, more manageable batches. Within dbt, these microbatches can often run at the same time (in parallel), enhancing efficiency.
+
+Parallel batch execution means that multiple batches are processed at the same time, instead of one after the other (sequentially). For example, if you have a micro-batch model with 12 batches, those batches can be executed in parallel. Specifically they'll run in parallel limited by the number of [available threads](docs/running-a-dbt-project/using-threads).
+
+To run batches in parallel you can use the `concurrent_batches` configuation:
+
+<File name='dbt_project.yml'>
+
+```yaml
+models:
+  +incremental_strategy: "concurrent_batches"
+```
+
+</File>
+
+or:
+
+<File name='models/my_model.sql'>
+
+```sql
+{{
+  config(
+    materialized='incremental',
+    incremental_strategy='concurrent_batches'
+    
+    ...
+  )
+}}
+
+select ...
+```
+
+</File>
+
+It's not possible to run batches concurrently when the model definition contains [`{{ this }}`](/reference/dbt-jinja-functions/this). This is because Jinja represents the state of a table.
+
+### Concurrent Execution with the `concurrent_batches` configuration
```

Comment:
what's the difference between this header ( Concurrent Execution with the `concurrent_batches` configuration) and the config example mentioned earlier? 

maybe this is more order of operations or  parallel execution sequence?